addon/HomepageStudy.jsm

```diff
@@ -71,19 +76,77 @@ function getHomepage() {
   return eTLD;
 }
 
+/**
+ * Returns the correct parameters depending if it's a standalone execution or a simulation.
+ * @param {boolean} isSimulation - Boolean indicating if the addon is run for a simulation.
+ * @param {string} rapporPath  - Path of the RAPPOR simulator.
+ */
+function getParams(isSimulation, rapporPath, instance) {
+  if (!isSimulation) {
+    return {filterSize: 16, numHashFunctions: 2, cohorts: 100, f: 0.0, p: 0.35, q: 0.65};
+  }
+
+  let params = Utils.read(new FileUtils.File(rapporPath + "_tmp/python/" + instance + "/case_params.csv"));
+  // In the file, the filterSize (k) value is in bits, but here we use bytes.
+  let filterSize = parseInt(params[1].split(",")[0], 10) / 8;
+  let numHashFunctions = parseInt(params[1].split(",")[1], 10);
+  let cohorts = parseInt(params[1].split(",")[2], 10);
+  let p = parseFloat(params[1].split(",")[3]);
+  let q = parseFloat(params[1].split(",")[4]);
+  let f = parseFloat(params[1].split(",")[5]);
+
+  return {filterSize: filterSize, numHashFunctions: numHashFunctions, cohorts: cohorts, f: f, p: p, q: q};
+}
+
+/**
+ * 
```

Comment:
Please describe what this function is supposed to do.

Reply:
Done